scipy/optimize/tests/test_linprog.py

```diff
@@ -2316,6 +2316,32 @@ def test_mip6(self):
 
         np.testing.assert_allclose(res.fun, 1854)
 
+    @pytest.mark.slow
+    @pytest.mark.timeout(120)
+    def test_mip_rel_gap_passdown(self):
```

Comment:
Solve the same problem as test_mip6, but scan over multiple `mip_rel_gap` values and check that the solution is solved to the correct optimality gap. Marked as slow and timeout because it should take strictly longer than `test_mip6` and that one is marked the same.

Reply:
It doesn't look like this showes that `mip_gap` is changing because of the option setting. For instance, `mip_gap` could be less than `0.001` even when  `mip_rel_gap = 0.5`, right? Can we show not just that `mip_gap < mip_rel_gap` but that `mip_rel_gap` is doing something?

(And not that you have to change anything, but would it be easy to show the same thing with a faster problem so that the test is not so slow?)

Reply:
Added an additional couple asserts that check that the differences between the reported mip_gaps are monotone decreasing, and not all zero as I crank down the requested mip_rel_gap.

Unfortunately, the problems in the faster tests are too simple, and the optimizer quickly finds a solution with primal-dual gap zero before it checks this termination criterion, so the differences between the reported mip_gaps are all zero in that case. Have to go with the slower test for now. 

Perhaps someone else can find a suitable problem (has to run quickly, but not be so simple that the optimizer quickly finds a solution with 0 primal-dual gap) here!

Reply:
That's ok. We don't need to hunt for a different problem. In just wanted to check the motivation for the slow problem.